xoinvader/ingame.py

```diff
@@ -251,9 +249,9 @@ def events(self):
 
     def update(self):
         self._collision_manager.update()
-        self.wave.update()
-        if not self.wave.running:
-            self.wave.start()
+        self.level.update()
```

Comment:
No. Level must not loop, it's quite obvious for me.
This little thing here is just for testing, and there's absolutely no need in complicating level inner logic for this minor case.